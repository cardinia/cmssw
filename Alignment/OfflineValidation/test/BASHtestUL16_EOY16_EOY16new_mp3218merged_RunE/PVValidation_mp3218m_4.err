26-Sep-2019 12:53:14 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/F0EE6221-5A07-E911-B26C-002590D9D9FC.root
26-Sep-2019 12:53:17 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/F0EE6221-5A07-E911-B26C-002590D9D9FC.root
Begin processing the 1st record. Run 276870, Event 5921402266, LumiSection 3449 on stream 0 at 26-Sep-2019 12:53:24.945 CEST
Begin processing the 1001st record. Run 276870, Event 5930010505, LumiSection 3455 on stream 0 at 26-Sep-2019 12:53:26.407 CEST
Begin processing the 2001st record. Run 276942, Event 3066311, LumiSection 11 on stream 0 at 26-Sep-2019 12:53:29.093 CEST
Begin processing the 3001st record. Run 276946, Event 2683605, LumiSection 3 on stream 0 at 26-Sep-2019 12:53:30.568 CEST
Begin processing the 4001st record. Run 276946, Event 11139754, LumiSection 10 on stream 0 at 26-Sep-2019 12:53:31.673 CEST
Begin processing the 5001st record. Run 276946, Event 18326520, LumiSection 16 on stream 0 at 26-Sep-2019 12:53:33.257 CEST
Begin processing the 6001st record. Run 276946, Event 26235090, LumiSection 22 on stream 0 at 26-Sep-2019 12:53:34.827 CEST
Begin processing the 7001st record. Run 277071, Event 270190708, LumiSection 145 on stream 0 at 26-Sep-2019 12:53:36.875 CEST
Begin processing the 8001st record. Run 277071, Event 278933239, LumiSection 149 on stream 0 at 26-Sep-2019 12:53:37.896 CEST
Begin processing the 9001st record. Run 277071, Event 288266976, LumiSection 154 on stream 0 at 26-Sep-2019 12:53:40.052 CEST
Begin processing the 10001st record. Run 277071, Event 294730552, LumiSection 158 on stream 0 at 26-Sep-2019 12:53:42.521 CEST
Begin processing the 11001st record. Run 277071, Event 301667720, LumiSection 162 on stream 0 at 26-Sep-2019 12:53:44.266 CEST
Begin processing the 12001st record. Run 277071, Event 310654749, LumiSection 168 on stream 0 at 26-Sep-2019 12:53:45.857 CEST
Begin processing the 13001st record. Run 277072, Event 817774832, LumiSection 462 on stream 0 at 26-Sep-2019 12:53:46.699 CEST
Begin processing the 14001st record. Run 277072, Event 824163220, LumiSection 466 on stream 0 at 26-Sep-2019 12:53:48.128 CEST
Begin processing the 15001st record. Run 277076, Event 1637393601, LumiSection 1028 on stream 0 at 26-Sep-2019 12:53:49.623 CEST
Begin processing the 16001st record. Run 277076, Event 1645158422, LumiSection 1033 on stream 0 at 26-Sep-2019 12:53:50.362 CEST
Begin processing the 17001st record. Run 277087, Event 1228014700, LumiSection 1137 on stream 0 at 26-Sep-2019 12:53:51.667 CEST
Begin processing the 18001st record. Run 277087, Event 1232040264, LumiSection 1141 on stream 0 at 26-Sep-2019 12:53:52.306 CEST
Begin processing the 19001st record. Run 277087, Event 1236795130, LumiSection 1145 on stream 0 at 26-Sep-2019 12:53:53.494 CEST
Begin processing the 20001st record. Run 277087, Event 1240814582, LumiSection 1149 on stream 0 at 26-Sep-2019 12:53:54.106 CEST
Begin processing the 21001st record. Run 277087, Event 1255522948, LumiSection 1162 on stream 0 at 26-Sep-2019 12:53:55.385 CEST
Begin processing the 22001st record. Run 277087, Event 1259162401, LumiSection 1166 on stream 0 at 26-Sep-2019 12:53:56.011 CEST
Begin processing the 23001st record. Run 277087, Event 1264379079, LumiSection 1171 on stream 0 at 26-Sep-2019 12:53:57.229 CEST
Begin processing the 24001st record. Run 277087, Event 1268857715, LumiSection 1175 on stream 0 at 26-Sep-2019 12:53:57.864 CEST
Begin processing the 25001st record. Run 277087, Event 1273306561, LumiSection 1179 on stream 0 at 26-Sep-2019 12:53:59.073 CEST
Begin processing the 26001st record. Run 277087, Event 1277649570, LumiSection 1183 on stream 0 at 26-Sep-2019 12:53:59.772 CEST
Begin processing the 27001st record. Run 277087, Event 1283021077, LumiSection 1188 on stream 0 at 26-Sep-2019 12:54:01.050 CEST
Begin processing the 28001st record. Run 277096, Event 2171269109, LumiSection 1244 on stream 0 at 26-Sep-2019 12:54:01.718 CEST
Begin processing the 29001st record. Run 277096, Event 2179207550, LumiSection 1249 on stream 0 at 26-Sep-2019 12:54:03.230 CEST
Begin processing the 30001st record. Run 277096, Event 2188127900, LumiSection 1255 on stream 0 at 26-Sep-2019 12:54:05.271 CEST
Begin processing the 31001st record. Run 277096, Event 2195852287, LumiSection 1260 on stream 0 at 26-Sep-2019 12:54:06.108 CEST
Begin processing the 32001st record. Run 277096, Event 2203577366, LumiSection 1265 on stream 0 at 26-Sep-2019 12:54:07.912 CEST
Begin processing the 33001st record. Run 277126, Event 21336049, LumiSection 48 on stream 0 at 26-Sep-2019 12:54:10.234 CEST
Begin processing the 34001st record. Run 277126, Event 29475609, LumiSection 58 on stream 0 at 26-Sep-2019 12:54:12.451 CEST
Begin processing the 35001st record. Run 277202, Event 795714887, LumiSection 509 on stream 0 at 26-Sep-2019 12:54:13.508 CEST
26-Sep-2019 12:54:13 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/F0EE6221-5A07-E911-B26C-002590D9D9FC.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 fileAction           -s file_close                             1        1
    2 fileAction           -s file_open                              2        2

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 fileAction           PostGlobalEndRun                  
    2 fileAction           pre-events       pre-events       

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
System                  3                   3

dropped waiting message count 0
[2.814MB/2.814MB][100%][==================================================][2.814MB/s]  [2.814MB/2.814MB][100%][==================================================][2.814MB/s]  
