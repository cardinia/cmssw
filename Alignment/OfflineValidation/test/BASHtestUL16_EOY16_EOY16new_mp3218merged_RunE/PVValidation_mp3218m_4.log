000 (3189561.004.000) 09/26 11:23:32 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3189561.004.000) 09/26 12:52:48 Started transferring input files
	Transferring to host: <188.184.186.97:9618?addrs=188.184.186.97-9618+[2001-1458-301-f8--100-53]-9618&noUDP&sock=4193_1904_13525>
...
040 (3189561.004.000) 09/26 12:52:48 Finished transferring input files
...
001 (3189561.004.000) 09/26 12:52:48 Job executing on host: <188.184.186.97:9618?addrs=188.184.186.97-9618+[2001-1458-301-f8--100-53]-9618&noUDP&sock=2393_fa39_3>
...
006 (3189561.004.000) 09/26 12:52:56 Image size of job updated: 35212
	21  -  MemoryUsage of job (MB)
	20516  -  ResidentSetSize of job (KB)
...
006 (3189561.004.000) 09/26 12:54:16 Image size of job updated: 502808
	21  -  MemoryUsage of job (MB)
	20516  -  ResidentSetSize of job (KB)
...
040 (3189561.004.000) 09/26 12:54:16 Started transferring output files
...
040 (3189561.004.000) 09/26 12:54:16 Finished transferring output files
...
005 (3189561.004.000) 09/26 12:54:16 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:00:40, Sys 0 00:00:02  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:40, Sys 0 00:00:02  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	7249  -  Run Bytes Sent By Job
	1015  -  Run Bytes Received By Job
	7249  -  Total Bytes Sent By Job
	1015  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :     2927        1   2126148 
	   Memory (MB)          :       21     2000      2000 
...
