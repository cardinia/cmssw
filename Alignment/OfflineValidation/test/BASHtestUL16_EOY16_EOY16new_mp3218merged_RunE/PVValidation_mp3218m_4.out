Job started at  Thu Sep 26 12:52:48 CEST 2019
batch dir: /pool/condor/dir_12733 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_276837_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 2.9M -rw-r--r--. 1 tkello zh 2.9M Sep 26 12:54 PVValidation_mp3218m_276837.root -rw-r--r--. 1 tkello zh 20K Sep 26 12:52 PVValidation_mp3218m_276837_cfg.py -rw-r--r--. 1 tkello zh 5.9K Sep 26 12:54 _condor_stderr -rw-r--r--. 1 tkello zh 522 Sep 26 12:53 _condor_stdout -rwxr-xr-x. 1 tkello zh 1015 Sep 26 12:52 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 26 12:53 tkello.cc drwx------. 3 tkello zh 4.0K Sep 26 12:52 tmp drwx------. 3 tkello zh 4.0K Sep 26 12:52 var
Job ended at  Thu Sep 26 12:54:16 CEST 2019
